docs/commands/index.md

```diff
@@ -5,22 +5,15 @@ tocMaxDepth: 2
 
 Redocly OpenAPI CLI currently supports the following commands:
 
-- [bundle](bundle.md)
-- [join](join.md)
-- [lint](lint.md)
-- [login](login.md)
-- [logout](logout.md)
-- [preview-docs](preview-docs.md)
-- [push](push.md)
-- [split](split.md)
-- [stats](stats.md)
+commands            |                                 | 
+--------------------|---------------------------------|------------------
+[bundle](bundle.md) | [split](split.md)               | [join](join.md)
+[lint](lint.md)     | [preview-docs](preview-docs.md) |  [stats](stats.md)
+[login](login.md)   | [logout](logout.md)             | [push](push.md)
```

Comment:
The more I look at this, the more I wonder about why not leave it as a bullet list?

Reply:
I agree here, the table doesn't make too much sense I believe as there is just a single dimension.

I think it just saves some vertical space.

Reply:
Seconded (or thirded :smile: ). To the readers, there's no clear or obvious reason why these commands are grouped in the table the way they are. 

If the columns had titles, that _might_ help a little...if the titles indicated the grouping criteria or some shared property of the commands listed in each column. 

However, that means that every new command we develop needs to fit into the table, and we'll have to spend extra time thinking where to put each new command so that the table still makes sense. That seems like too much cost for little benefit, so it would be easier to just leave it as a plain alphabetically sorted list.